src/App.tsx

```diff
@@ -15,36 +16,90 @@
   'Garlic',
 ];
 
+const SORT_FILD_ALPHABETICCALY = 'alphabeticcaly';
+const SORT_FILD_LENGTH = 'length';
+
+function getPreparedGoods(goods: string[], sortField: string, rever: boolean) {
```

Comment:
The parameter `rever` seems to be a typo. It should be `reverse` to clearly indicate its purpose.